Model/Ui/CapabilitiesConfigProvider.php

```diff
@@ -47,6 +48,12 @@ public function getConfig()
                     $backendsFpx = $this->capabilities->getBackendsByType(Fpx::TYPE);
                     $configs['fpx']['banks'] = $backendsFpx ? current($backendsFpx)->banks : [];
                     break;
+                case Mobilebanking::CODE:
+                    $configs['mobile_banking'] = $this->capabilities->retrieveMobileBankingBackends();
+                    break;
+                case Internetbanking::CODE:
+                    $configs['internet_banking'] = $this->capabilities->getBackendsByType("internet_banking");
```

Comment:
why don't we reuse the function `getBackendsByType`  for mobile banking too? just wondering if there was a reason

Reply:
Is bc our omsie-php is using `aegaeon` version for capabilities and we set is like this
``` ruby
    "internet_banking_bay" => "internet_banking",
    "internet_banking_ktb" => "internet_banking",
    "internet_banking_scb" => "internet_banking",
    "internet_banking_bbl" => "internet_banking",
    "alipay" => "alipay",
    "installment_bay" => "installment",
    "installment_kbank" => "installment",
    "installment_ktc" => "installment",
    "installment_bbl" => "installment",
    "installment_first_choice" => "installment",
    "installment_citi" => "installment",
    "installment_scb" => "installment",
    "installment_ttb" => "installment",
    "installment_uob" => "installment",
    "installment_ezypay" => "installment",
    "mobile_bป--anking_scb" => "mobile_banking_scb",
    "mobile_banking_kbank" => "mobile_banking_kbank",
    "mobile_banking_bbl" => "mobile_banking_bbl",
    "mobile_banking_bay" => "mobile_banking_bay",
    "mobile_banking_ocbc_pao" => "mobile_banking_ocbc_pao",
   ```

Reply:
ohhh I see, that makes sense now thanks for explaining 🙂